src/OpenDiffix.Core/Analyzer.fs

```diff
@@ -423,18 +423,23 @@ let rec private normalizeBucketLabelExpression expression =
     FunctionExpr(ScalarFunction fn, List.map normalizeBucketLabelExpression args @ extraArgs)
   | _ -> expression
 
-let private untrustedAllowsRange arg =
+let private isMoneyStyle arg =
   match arg with
   // "money-style" numbers, i.e. 1, 2, or 5 preceeded by or followed by zeros: ⟨... 0.1, 0.2, 0.5, 1, 2, 5, 10, 20, ...⟩
   | Constant (Real c) -> Regex.IsMatch($"%.15e{c}", "^[125]\.0+e[-+]\d+$")
   | Constant (Integer c) -> Regex.IsMatch($"%i{c}", "^[125]0*$")
   | _ -> false
 
-let private validateBucketLabelExpression accessLevel expression =
+let private validateGeneralization accessLevel expression =
```

Comment:
I think `validateBucketLabelExpression` is the more general term.

Reply:
So I thought initially, but then I observed this is actually that specific. Note that we also do other kinds of bucket label expression validations. Once untrusted analyst mode concerns anything else than generalizations, we'll rename/refactor as needed. OTOH no-one wiill get the wrong idea that this functions _is the_ bucket label expression validation routine.